tests/e2e/support/page/publicLinkPage.ts

```diff
@@ -0,0 +1,37 @@
+import { BrowserContext, Page } from 'playwright'
+import { state } from '../../cucumber/environment/shared'
+import { lastCreatedPublicLink } from './files/publicLink'
+import { expect, Locator } from '@playwright/test'
+
+export class PublicLinkPage {
+  public context: BrowserContext
```

Comment:
There's `publickLink` page in `.../page/files`. Can these go there?

Reply:
yes it can go there